src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceNamer.java

```diff
@@ -198,4 +199,14 @@ public String getReleaseAnnotation(ReleaseLevel releaseLevel) {
   public String getBatchingDescriptorConstName(Method method) {
     return inittedConstantName(Name.upperCamel(method.getSimpleName()).join("batching_desc"));
   }
+
+  @Override
+  public String getPackagePath() {
+    List<String> packagePath = Splitter.on(".").splitToList(getPackageName());
+    if (packagePath.size() > 3) {
+      // Remove the "spi.{version}" suffix
+      return Joiner.on("/").join(packagePath.subList(0, packagePath.size() - 2));
```

Comment:
We should check to make sure our assumption is correct (that the package actually has the spi.version suffix), and if not, either throw an exception or do something different.